src/main/webapp/synchronizer.js

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 let videoSyncTimer;
-let lastTime;
+let lastTimeMs;
```

Comment:
Can you rename this to something like `lastSyncedTime`?

Reply:
Done